homeassistant/components/ambient_station/strings.json

```
@@ -10,11 +10,11 @@
       }
     },
     "error": {
-      "invalid_key": "Invalid API Key and/or Application Key",
+      "invalid_key": "[%key:common::config_flow::error::invalid_api_key%]",
       "no_devices": "No devices found in account"
```

```suggestion
      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
```
Network or account kind of debatable.

---

For me accounts sounds like cloud thing but device sounds local thing.

---

`no_devices_found` reference string translates to local network, which is not the same as the current meaning here. Let's leave this as-is for now.